assets/javascripts/discourse/initializers/apply-tabs.js.es6

```diff
@@ -0,0 +1,116 @@
+import {withPluginApi} from 'discourse/lib/plugin-api';
+import ComposerController from 'discourse/controllers/composer';
+
+function initializeTabs(api) {
+  api.decorateCooked(
+    $elem => {
+      const radTabsElements = $elem.find(`.js-rad-tabs-element`);
+
+      radTabsElements.each(function() {
+        const radTabsElement = $(this);
+        const tabs = radTabsElement.find(`.js-rad-content-element`);
+        const header = $(`<div class='p-code-snippet__header'></div>`);
+        const dropdowns = $(`<div class='p-code-snippet__dropdowns'></div>`);
+
+        radTabsElement.addClass('p-code-snippet');
+
+        const dropdownSettings = getDropdownSettings(tabs);
+        for (let i = 0; i < dropdownSettings.length; i++) {
```

Comment:
As far as I know, `forEach` in JavaScript is a much faster loop than `for loops`. And you don't seem to be manipulating index anywhere, so I think the `forEach((element) => ...)` can save you from using the `[i]` to access the element.

Reply:
Updated multiple `for`s to `forEach`. Good suggestion.